src/main/java/com/dabsquared/gitlabjenkins/service/GitLabProjectBranchesService.java

```diff
@@ -34,9 +35,9 @@ public static GitLabProjectBranchesService instance() {
         return gitLabProjectBranchesService;
     }
 
-    public List<String> getBranches(GitLabClient client, String sourceRepositoryString) {
+    public List<String> getBranches(GitLabApi gitLabApi, String sourceRepositoryString) {
```

Comment:
Here and elsewhere it is creating a lot of diff churn to rename both the type from GitLabClient to GitLabApi and the variable from client to gitLabApi. Why don't you leave the variable named `client` to cut down on diff churn. It is not inaccurate: `GitLabApi` _is_ an API client. And this will cut down on the size of an already very large diff.